Examples_Nuget/QuickstartUdf/Program.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using TileDB.Cloud;
+using TileDB.Cloud.Rest.Model;
+
+namespace QuickstartUdf
+{
+    class Program
+    {
+        static UDFInfo UdfGetInfo(string uri)
+        {
+            return TileDB.Cloud.RestUtil.Udf.GetInfo(uri);
+        }
+
+        // Run generic UDF using JSON to pass args
+        // + (Python) UDF declaration for this example: def args_udf(arg1, arg2="default")
+        static void UdfGenericExec(string udfUri, string chargedOrg=null, string args=null)
+        {
+            Console.WriteLine($"Running generic UDF {udfUri}...");
+            DebugOutput(udfUri, chargedOrg, args);
+
+            var result = TileDB.Cloud.RestUtil.Udf.ExecGeneric(udfUri, args, chargedOrg: chargedOrg);
+            // Output result to console
+            Console.Write("Result from generic UDF: ");
+            result.CopyTo(Console.OpenStandardOutput());
+            Console.WriteLine("\n");
+        }
+
+        static void UdfGenericExecAsync(string udfUri)
+        {
+            Console.WriteLine($"Running generic UDF {udfUri} asynchronously...");
+            // Call UDF asynchronously; Does not block execution on call to SubmitGenericUDFAsync()
+            var resultTask = TileDB.Cloud.RestUtil.Udf.ExecGenericAsync(udfUri, chargedOrg: "TileDB-Inc");
+            // Here we are free to do some work while waiting on resultTask...
+            Console.WriteLine("Started async UDF task...");
+
+            Console.Write($"Async UDF task completed with output: ");
+            // Accessing the `Result` property of `taskResponse` will block until async task completes
+            resultTask.Result.Result.CopyTo(Console.OpenStandardOutput());
```

Comment:
Why do we use `CopyTo` rather than `Console.WriteLine` here?

Reply:
The return value of generated OpenAPI methods is `System.IO.Stream` which can't be used with any `Console.WriteLine` overload directly. Streams support copying to other streams though, and since standard out is a stream we can just copy to stdout to print the message.

I can encapsulate this return value and return a string object instead, which would probably be more intuitive for the client. For synchronous requests this would be NBD, but for async I may need to add another layer of encapsulation since the (task-wrapped) return value of [ExecGenericAsync](https://github.com/TileDB-Inc/TileDB-Cloud-CSharp/blob/4812a7bec7eb425e873b5191f8787c67a5b56e59/TileDB.Cloud/Udf.cs#L60) is determined by the OpenAPI method [SubmitGenericUDFAsync](https://github.com/TileDB-Inc/TileDB-Cloud-CSharp/blob/4812a7bec7eb425e873b5191f8787c67a5b56e59/TileDB.Cloud.Rest/src/TileDB.Cloud.Rest/Api/UdfApi.cs#L1932)

Reply:
Makes sense, thanks.